### DISCUSSION TITLE
为啥discussions，issue，PR都找不到#1128……

### DISCUSSION THREAD

**Discussion by Deep-Dark-Forest** [Meloong-Git/PCL#2948] [提问] [answered]
这个issue/pr/discussion到底发生了啥？

**Comment by burningtnt** [comment]
被删了而已

**Reply by burningtnt** [reply]
比如，如果一个人提交 Issue 过快，新的 Issue 就会被删除

**Reply by Deep-Dark-Forest** [reply]
> 比如，如果一个人提交 Issue 过快，新的 Issue 就会被删除

好像没这么简单？

**Reply by MoYuan-CN** [reply]
找不到会有以下几种可能
1. 暂缓处理转处理中，被删了（不过能在某条 Issue 正文里找到）
2. 内容不雅或提交过快被删了
3. √ 涉及加密内容等有关启动器安全性问题，被删了（见下）

**Comment by WForst-Breeze** [comment, marked as answer]
这个 Issue 有关隐藏主题的破解，反馈者在提交反馈后直接公开了破解方式，鉴于涉及到保密内容于是被删除。但此问题已修复，破解方式已失效

![image](https://github.com/Hex-Dragon/PCL2/assets/110760354/0c739518-b21b-483d-9c3d-cd981b0f0c64)

**Reply by SqrtTan** [reply]
这种东西是不是应该在security里交，那地方看不到

**Reply by MoYuan-CN** [reply]
理论上是的，不过还是有人把他送进了 Issue 里..
另外这个帖的 Author 好像是故意交到了 Issue 里

**Reply by SqrtTan** [reply]
不安好心了只能说

**Reply by MoYuan-CN** [reply]
en

**Comment by burningtnt** [comment]
？
![image](https://github.com/Hex-Dragon/PCL2/assets/88144530/06e759cb-b38a-4182-a54f-d63a87071441)
这个理由，笑死了

**Comment by iPodtouch6557** [comment]
so原作者是谁
